docs/how-to/enabling-logging.rst

```diff
@@ -0,0 +1,140 @@
+.. meta::
+  :description: Enabling logging in rocFFT
+  :keywords: rocFFT, ROCm, API, documentation, logging
+
+
+.. _enabling-logging:
+
+********************************************************************
+Enabling logging in rocFFT
+********************************************************************
+
+rocFFT can write a variety of log messages to aid troubleshooting. Here are the different
+logs that rocFFT supports.
+
+*  **Trace logging**: Logs the library entry points (for example, ``rocfft_plan_create`` or ``rocfft_execute``) and their parameter
+   values when they are called. Error messages during plan creation and execution are also logged here.
+*  **Bench logging**: Logs the ``rocfft-rider`` command line when a plan is created.
+   You can use this command to rerun the same transform again later.
+*  **Profile logging**: Logs a message for each kernel launched during plan execution.
+   This message contains the following elements:
+
+   *  Kernel duration
+   *  The size of the user data buffers seen by the kernel
+   *  Estimates for the observed memory bandwidth and bandwidth efficiency
+
+   .. note::
+   
+      To provide the kernel duration, rocFFT must use ``hipEvents`` and wait for each kernel to complete.
+      This might interfere with time measurement at higher levels, for example, for ``rocfft-rider``.
```

Comment:
`rider` -> `bench`.  `rocfft-rider` was the older and less meaningful name for the benchmark tool.